tests/pytests/test_issues.py

```diff
@@ -169,3 +169,26 @@ def test_issue1988(env):
     env.expect('FT.SEARCH', 'idx', 'foo', 'WITHSCORES').equal([1L, 'doc1', '1', ['t', 'foo']])
     env.expect('FT.SEARCH', 'idx', 'foo', 'SORTBY' , 't').equal([1L, 'doc1', ['t', 'foo']])
     env.expect('FT.SEARCH', 'idx', 'foo', 'WITHSCORES', 'SORTBY' , 't').equal([1L, 'doc1', '1', ['t', 'foo']])
+
+def testIssue2104(env):
```

Comment:
Should we also add test in test_json.py, with `APPLY`, when name (attribute) and also path (identifier) can be `$.n`, `.n` or `n`?
(with the format `{identifier} [AS {attribute}]`)